protocol/contracts/amo/AuraStaking.sol

```diff
@@ -107,8 +120,9 @@ contract AuraStaking is Ownable {
         uint256 depositTokenBalance = AMO__IBaseRewardPool(auraPoolInfo.rewards).balanceOf(address(this));
         AMO__IBaseRewardPool(auraPoolInfo.rewards).withdrawAllAndUnwrap(claim);
         if (sendToOperator) {
-            // unwrapped amount is 1 to 1
-            bptToken.safeTransfer(operator, depositTokenBalance);
+            // Include any BPT balance in this contract
+            uint256 totalBalance = depositTokenBalance + bptToken.balanceOf(address(this));
```

Comment:
I think depositTokenBalance is double counted here, since L121 will withdraw the bpt tokens and increase the contract's current balance

Reply:
How so sorry?

This is withdrawing everything, so:
1/ withdraw everything from aura
2/ transfer everything we claimed from aura (depositTokenBalance), plus any other BPT sitting in this contract